apps/tasks.py

```diff
@@ -636,3 +636,14 @@ def purge_tasks():
     Remove tasks from queue to avoid overflow
     """
     app.control.purge()
+
+
+@app.task
+def delete_old_objects():
+    # Define the time threshold (24 hours ago)
+    threshold_time = timezone.now() - timezone.timedelta(days=7)
```

Comment:
Seems like a couple of lines of docstrings would be nice

I would also like to suggest making this configurable via settings.py .

Reply:
Great suggestion. Ill add docstring. 
Regarding the settings, i agree but maybe a different PR since this must update the information regarding this plus the charts repo. Agree?

Reply:
Docstring added